content/_includes/ClusterTypes.md

```diff
@@ -1,9 +1,19 @@
-* **Replicated** - Replicate files across bricks in the volume. 
-You can use replicated volumes in environments where high availability and high reliability are critical.
-* **Distributed Replicated** - Distribute files across replicated bricks in the volume. 
-  You can use distributed replicated volumes in environments where you need to scale storage and high reliability is critical. 
-  Distributed replicated volumes also offer improved read performance in most environments. Requires setting an additional **Replica Count**.
-* **Dispersed** - Dispersed volumes are based on erasure codes, providing space-efficient protection against disk or server failures. 
-  It stores an encoded fragment of the original file in each brick so that only a subset of the fragments are needed to recover the original file. 
-  When creating the volume, the administrator configures the number of bricks that can be missing without losing access to data. 
-  Choosing Dispersed requires setting an additional **Redundancy Count**.
\ No newline at end of file
+### Replicated
+
+Replicated volumes are the most similar to ZFS mirrors. They have exact copies of all data on all bricks. Since TrueNAS SCALE’s SMB cluster implementation requires a minimum of three nodes, a replicated volume will have three identical copies of all data. 
```

Comment:
don*'t use a possessive and no future tense, use present tense
```suggestion
Replicated volumes are the most similar to ZFS mirrors. They have exact copies of all data on all bricks. Since TrueNAS SCALE SMB cluster implementation requires a minimum of three nodes, a replicated volume has three identical copies of all data. 
```